source/extensions/omni.isaac.orbit/omni/isaac/orbit/sim/utils.py

```diff
@@ -777,3 +777,47 @@ def find_global_fixed_joint_prim(
                 break
 
     return fixed_joint_prim
+
+
+"""
+USD Variants.
+"""
+
+
+def set_usd_variants(prim_path: str, variants: object | dict[str, str], stage: Usd.Stage | None = None) -> None:
+    """
+    Sets the variant selections for the specified variant sets on a USD prim.
```

Comment:
It would help a lot here to have more documentation on variants and what that means. You can refer to this function in the USDFileCfg configclass object to avoid duplication in docstrings for variants.

Reply:
Added a link to the official USD documentation. Let me know if you think more is necessary. 
Imo, people will mostly only use this function if they already know what Variants are, so I wouldn't want to write a lengthy explanation here.